deploy/commands.py

```diff
@@ -34,8 +35,8 @@ def get_sandboxes(args):
     return sandboxes
 
 def get_pod_statuses(args):
-    output = run_script(
-        ['kubectl', 'get', 'pods', '--all-namespaces', '--output=json'], args)
+    output = run_(
+        ['kubectl', 'get', 'pods', '--all-namespaces', '--output=json'])
```

Comment:
did you intentionally or accidentally remove the args here? How are you using them now?

Reply:
intentionally - the args/env variables are not needed by the kubectl commands.